docs/docs/tutorials/multi_agent/hierarchical_agent_teams.ipynb

```diff
@@ -291,7 +291,16 @@
     "from langchain_openai import ChatOpenAI\n",
     "\n",
     "from langgraph.graph import END, StateGraph, START\n",
-    "from langchain_core.messages import HumanMessage\n",
+    "from langchain_core.messages import HumanMessage, trim_messages\n",
+    "\n",
+    "llm = ChatOpenAI(model=\"gpt-4o-mini\")\n",
+    "\n",
+    "trimmer = trim_messages(\n",
+    "    max_tokens=100000,\n",
```

Comment:
ooc -- does it not work without the trimmer?

Reply:
context window gets exceeded